stdlib/public/Concurrency/PartialAsyncTask.swift

```diff
@@ -194,9 +235,16 @@ internal func _resumeUnsafeThrowingContinuationWithError<T>(
 
 #endif
 
-/// The operation functions must resume the continuation *exactly once*.
+/// Calls the given closure with an unsafe continuation
+/// and returns its result.
 ///
-/// The continuation will not begin executing until the operation function returns.
+/// - Parameter fn: A closure that takes an `UnsafeContinuation` parameter.
+/// The closure must resume the continuation exactly once.
+///
+/// - Returns: The value passed to the continuation by the closure.
+///
+/// The continuation won't start executing until the operation function returns.
+/// ◊TR: What does this mean?
```

Comment:
I don't know what this means. I think the summary text for the `withChecked*` forms, with my suggested edits, could be reused here, just adjusted to note the unsafety and lack of checking of these forms.

Reply:
Thanks. I think the change in 55e8ec1fdc8 to the abstracts was likely what this text was trying to call out — the fact that calling a "with continuation" method is a suspension point.